pkg/olm/clusterserviceversion.go

```diff
@@ -120,7 +120,44 @@ func (builder *ClusterServiceVersionBuilder) GetAlmExamples() (string, error) {
 		}
 	}
 
-	return "", fmt.Errorf("%s not found in given csv named %v", almExamples, builder.Definition.Name)
+	return "", fmt.Errorf("%s not found in given clusterServiceVersion named %v",
+		almExamples, builder.Definition.Name)
+}
+
+// IsSucceeded check if the clusterServiceVersion is Succeeded.
```

Comment:
checks

Reply:
Also, I'd rename the function to isSuccessful for readability and update the comments with is Succeeded to "is Successful"